java/vertx4/config.yaml

```
@@ -0,0 +1,14 @@
+framework:
+  website: https://vertx.io/docs/vertx-core/java/
```

no need to have `https://`
```suggestion
  website: vertx.io/docs/vertx-core/java/
```

---

URLs should have schemes.